Apps/WebClient/src/ClientApp/app/components/timeline/medication.vue

```diff
@@ -235,6 +259,24 @@ export default class MedicationTimelineComponent extends Vue {
     return faPills;
   }
 
+  private toggleComments(): void {
+    this.commentsVisible = !this.commentsVisible;
+    this.hasErrors = false;
+    if (!this.commentsVisible) {
+      return;
+    }
+  }
+
+  private sortComments() {
+    this.comments.sort((a, b) =>
+      a.createdDateTime > b.createdDateTime
```

Comment:
NIT:  This is probably clearer as an if/else especially since the formatting is on just as many lines.

Reply:
Changes made